test/jira-tests.js

```diff
@@ -30,11 +30,7 @@ describe('Jira API Tests', () => {
     });
 
     it('Constructor with no auth credentials', () => {
-      const {
-        username,
-        password,
-        ...options
-      } = getOptions();
+      const { username, password, ...options } = getOptions(); // eslint-disable-line no-use-before-define, max-len
```

Comment:
I do not.  I should have linked it when I was researching it and I have no idea what possessed me to not do that.  When I update eslint again on this branch, I'll remove those eslint overrides and see if they're fixed.  But right now, to avoid the other major problem with eslint, I'd need to update to a beta version of babel-6.0.0 and eslint 2.4.0 at the same time.  The only other solution is to stay  on 2.2.0 for now, according to this: https://github.com/babel/babel-eslint/issues/267